tests/XrdCks/XrdCksTest.sh

```
@@ -0,0 +1,16 @@
+
+# The first argument designates the type of test being run and should
+# be either "demand" or "r/t". We set a default here.
+test="${1:-unknown}"
+
+# When the test is for demand checksums, the server should be started
+# with the XrdCksTestOD.cf config file.
+
+# When the test is for r/t checksums, the server should be started
+# with the XrdCksTestRT.cf config file.
+#
+./TestCheckSum $test adler32 64fce8a2 XrdCksTestFile $target
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)